### PR TITLE
feat(pnp): add paginated requests handler and use address/name vs. ow…

### PR DESCRIPTION
**Pull request by dtfiedler**
…ner/name

### CODE REVIEW

File: src/primary_names.lua [hunk 1]
```diff
@@ -300,7 +300,7 @@ end
 function primaryNames.getPaginatedPrimaryNames(cursor, limit, sortBy, sortOrder)
 	local primaryNamesArray = {}
 	local cursorField = "name"
-	for owner, primaryName in ipairs(primaryNames.getUnsafePrimaryNameOwners()) do
+	for owner, primaryName in pairs(primaryNames.getUnsafePrimaryNameOwners()) do
```
Comment (dtfiedler) [inline src/primary_names.lua:303]: 🐛

File: tests/primary.test.mjs [hunk 2]
```diff
@@ -412,4 +412,29 @@ describe('primary names', function () {
...
+          { name: 'Sort-Order', value: 'asc' },
+        ],
+      });
+      assertNoResultError(getPaginatedPrimaryNameRequestsResult);
+      const primaryNameRequests = JSON.parse(
+        getPaginatedPrimaryNameRequestsResult.Messages[0].Data,
+      );
+      assert.deepStrictEqual(primaryNameRequests, {
+        items: [],
+        totalItems: 0,
```
Comment (arielmelendez) [inline tests/primary.test.mjs:432]: I can haz 1 item and a cursor plz?

Comment (arielmelendez) [reply tests/primary.test.mjs:432]: Unit tests look alright. Might be nice to see cursor and a sortBy preference exercised here though.

**Review by arielmelendez — APPROVED** [review]
Looks clean. Tests could use a bit more scope but these are pretty well worn patterns.

**Pull request merged**